frontend/src/components/common/body-wrapper/BodyWrapper.tsx

```diff
@@ -1,8 +1,9 @@
-import React from 'react'
 import cx from 'classnames'
 
 import styles from './BodyWrapper.module.scss'
 
+import type { FunctionComponent } from 'react'
+
```

Comment:
To follow the import order convention, external module imports should be at the top. We could also look into if we want to use this plugin to sort import order 
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order

Reply:
Thanks! I agree about sorting the import order with a plugin. I've looked into both `eslint-plugin-import` and `eslint-plugin-simple-import-sort` and settled on the latter since it seems to do a better job at ordering the imports out of the box.